docs/dev/developer-guides/transactions/blocks.md

```diff
@@ -58,11 +63,35 @@ To avoid this is why we built a purely fictional block containing the event o
 Block hashes in zkSync are deterministic and are derived from the following formula: "keccak256(l2_block_number)".
 The reason for having a deterministic block hash is that these hashes are not provable (remember that L2 blocks are not submitted to L1).
 Projects are advised not to use the L2 block hash as a source of randomness.
+## Block Properties
+
+## block.number and block.timestamp
+
+Depending on where it's used, if `block.number` and `block.timestamp` is retrieved via the `zksync-web3` SDK, mini blocks values are returned. It is a special construct on zkSync Era to make sure that the transactions are processed promptly. This approach is similar in other L2s, where each transaction belongs to a separate block, (which we call mini blocks), while these mini blocks are gathered together into batches that get sent to L1.
```

Comment:
Just in case, did we decide to call "mini blocks" and not "miniblocks", right? (I personally prefer "miniblocks", but if this is something that has been decided I don't mind)

Reply:
👍